src/Internal/Payment/State/InitialState.php

```diff
@@ -46,69 +42,53 @@ class InitialState extends AbstractPaymentState {
 	 *
-	 * @param StateFactory                 $state_factory           Factory for payment states.
-	 * @param OrderService                 $order_service           Service for order-related actions.
-	 * @param WC_Payments_Customer_Service $customer_service        Service for managing remote customers.
-	 * @param Level3Service                $level3_service          Service for Level3 Data.
-	 * @param PaymentRequestService        $payment_request_service Connection with the server.
+	 * @param StateFactory                 $state_factory Factory for payment states.
+	 * @param OrderService                 $order_service Service for order-related actions.
+	 * @param WC_Payments_Customer_Service $customer_service Service for managing remote customers.
+	 * @param Level3Service                $level3_service Service for Level3 Data.
 	 */
 	public function __construct(
 		StateFactory $state_factory,
 		OrderService $order_service,
 		WC_Payments_Customer_Service $customer_service,
-		Level3Service $level3_service,
-		PaymentRequestService $payment_request_service
+		Level3Service $level3_service
 	) {
 		parent::__construct( $state_factory );
 
-		$this->order_service           = $order_service;
-		$this->customer_service        = $customer_service;
-		$this->level3_service          = $level3_service;
-		$this->payment_request_service = $payment_request_service;
+		$this->order_service    = $order_service;
+		$this->customer_service = $customer_service;
+		$this->level3_service   = $level3_service;
 	}
 
 	/**
 	 * Initialtes the payment process.
 	 *
-	 * @param PaymentRequest $request    The incoming payment processing request.
-	 * @return CompletedState            The next state.
+	 * @param PaymentRequest $request The incoming payment processing request.
+	 *
+	 * @return AbstractPaymentState      The next state.
 	 * @throws StateTransitionException  In case the completed state could not be initialized.
 	 * @throws ContainerException        When the dependency container cannot instantiate the state.
 	 * @throws Order_Not_Found_Exception Order could not be found.
 	 * @throws PaymentRequestException   When data is not available or invalid.
 	 */
-	public function process( PaymentRequest $request ) {
-		$context  = $this->get_context();
-		$order_id = $context->get_order_id();
-
+	public function start_processing( PaymentRequest $request ) {
 		// Populate basic details from the request.
 		$this->populate_context_from_request( $request );
 
 		// Populate further details from the order.
 		$this->populate_context_from_order();
 
-		// Payments are currently based on intents, request one from the API.
-		try {
-			$intent = $this->payment_request_service->create_intent( $context );
-		} catch ( Invalid_Request_Parameter_Exception | Extend_Request_Exception | Immutable_Parameter_Exception $e ) {
-			return $this->create_state( SystemErrorState::class );
-		}
+		/**
+		 * Next state variable.
+		 *
+		 * @var VerifiedState $next_state
```

Comment:
Instead of defining this here, please make sure that all transition methods (`start_processing`, `process`, and `complete`) are present in the abstract state.